src/README.md

```diff
@@ -73,7 +73,18 @@ juju config neutron-api flat-network-providers="physnet1 physnet2"
 juju config neutron-gateway bridge-mappings="physnet1:br-ex physnet2:br-trove" data-port="br-ex:eth2 br-trove:eth3"
 juju config neutron-openvswitch bridge-mappings="physnet2:br-trove" data-port="br-trove:eth2"
 
-# Define the flat OpenStack network and subnet.
+# After the charm has been deployed, define the flat OpenStack network, and subnet
+# with a route to the Management Network.
+juju run-action trove/leader create-management-network physical-network=physnet2 \
```

Comment:
This can also be a `VLAN` network type. In some cases, I suspect other network types might also work. So it may be useful to add `net_type` and `segmentation_id` as options.

Reply:
Added. For the moment, I've only added flat and vlan as options, but that can easily be removed.